flink-table-store-core/src/main/java/org/apache/flink/table/store/file/operation/FileStoreReadImpl.java

```diff
@@ -44,6 +44,7 @@
     private final Accumulator accumulator;
 
     private boolean keyProjected;
+    private boolean dropDelete = true;
```

Comment:
why directly set `dropDelete` to `true`, when `FileStoreReadImpl.<init>`, it will be assign to `false` anyway.